docs/querying/api.md

```diff
@@ -653,6 +653,52 @@ curl -G http://localhost:9091/api/v1/targets/metadata \
 }
 ```
 
+## Querying metric metadata
+
+Similarly to the previous endpoint `api/v1/targets/metadata`, it returns metadata about metrics currently scrapped by targets. However, it does not provide any target information.
+This is considered **experimental** and might change in the future.
+
+```
+GET /api/v1/metadata
+```
+
+URL query parameters:
+
+- `limit=<number>`: Maximum number of metrics to return.
+
+The `data` section of the query result consists of an object where each key is a metric name and each value is a list of unique metadata objects, as exposed for that metric name across all targets.
+
+The following example returns two metrics. Note that the metric `http_requests_total` has more than one object in the list. At least one target has a value for `HELP` that not match with the rest.
```

Comment:
that did not

Reply:
Sorry, not sure I follow the comment. I can't spot the mistake 🤦‍♂.

Reply:
Negations have to happen with "do not" in English instead of just adding a "not" to the full verb like in all other languages. :o)

"that not match with the rest." has to be "that do not match with the rest."

Reply:
🤦‍♂ , sorry - I read the text over 10 times and kept adding that do in my mind all time.